packages/dashboard/src/components/Card/Chart/Chart.tsx

```diff
@@ -5,20 +5,56 @@ import { LineChart } from '@uifabric/charting/lib/LineChart';
 import { VerticalBarChart } from '@uifabric/charting/lib/VerticalBarChart';
 import { DonutChart } from '@uifabric/charting/lib/DonutChart';
 import { PieChart } from '@uifabric/charting/lib/PieChart';
-import { StackedBarChart } from '@uifabric/charting/lib/StackedBarChart';
+import {
+  StackedBarChart,
+  IDataPoint,
+  ILegendDataItem,
+  MultiStackedBarChart
+} from '@uifabric/charting/lib/StackedBarChart';
 
 export class Chart extends React.Component<IChartInternalProps, {}> {
+  public static defaultProps = {
+    compactChartWidth: 250
+  };
+
+  private _chartLabel: string | undefined;
+  private _isMultiBarChart = false;
+  private _colors: string[] | undefined;
+  private _singleChartDataPoints: IDataPoint[] | undefined;
+
+  public constructor(props: IChartInternalProps) {
+    super(props);
+
+    if (props.chartLabels && props.chartLabels.length === 1) {
+      this._chartLabel = props.chartLabels[0];
+    }
+
+    if (props.data) {
+      this._isMultiBarChart = props.data.length > 1;
+    }
+
+    this._singleChartDataPoints = props.dataPoints;
+
+    if (this._isMultiBarChart === false && props.data && props.data.length > 0) {
```

Comment:
>his._isMultiBarChart === false [](start = 9, length = 30)

!this._isMultiBarChart?

Reply:
just being explicit

---
In reply to: [209399127](https://github.com/OfficeDev/office-ui-fabric-react/pull/5906#discussion_r209399127) [](ancestors = 209399127)